diesel/Cargo.toml

```diff
@@ -19,6 +19,7 @@ chrono = { version = "^0.2.17", optional = true }
 uuid = { version = ">=0.2.0, <0.4.0", optional = true, features = ["use_std"] }
 
 [dev-dependencies]
+diesel_codegen = { path = "../diesel_codegen" }
```

Comment:
Can you add this with the current version and add an override in the root-level Cargo.toml? Otherwise crates.io won't let us publish this (IIRC).